src/adapters/superagent.js

```diff
@@ -0,0 +1,43 @@
+import superagent from 'superagent';
+import * as httpMethods from '../constants/http-methods';
+
+export const __createRequest = (url, method) => {
```

Comment:
Is this to indicate that it shouldn't be used except by tests? If so I think it's fine to just export it without the `__` prefix.